lib/modules/asset/AssetsController.ts

```diff
@@ -37,6 +38,12 @@ export class AssetsController {
           handler: this.create.bind(this),
           http: [{ path: "device-manager/:engineId/assets", verb: "post" }],
         },
+        upsert: {
+          handler: this.upsert.bind(this),
+          http: [
+            { path: "device-manager/:engineId/assets/:_id", verb: "post" },
```

Comment:
This path does not reflect the behaviour of this action

Reply:
I used POST by following this : https://docs.kuzzle.io/core/2/api/controllers/document/upsert/
Should I change it to PUT?

Reply:
Nah, you are right I though the path was the same as the update action, but nvm it's okay :)

Reply:
no problem !